app/classifier/classifier.jsx

```diff
@@ -239,7 +246,15 @@ class Classifier extends React.Component {
           onChange={this.handleAnnotationChange.bind(this, currentClassification)}
           playIterations={this.props.workflow.configuration.playIterations}
         />
-
+        {this.props.workflow.configuration.metadata &&
+          this.props.workflow.configuration.metadata.type === 'modelling' ?
+            <ModelCanvas
```

Comment:
This is an instance of the `ModelRenderer` component, so the name is a bit confusing.